Dockerfiles/agent/entrypoint/50-ci.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash
set -e
```

Reply:
makes sense, should we add this to all entrypoint shell scripts as well? (none of them have this at the moment, not sure if it's set at a higher level)

Reply:
I was looking at where we exec `/etc/cont-init.d/` scripts: https://github.com/DataDog/datadog-agent/blob/fe33f55ae3ef1f63b9b70eadb4e71d1d408b77dc/Dockerfiles/manifests/agent-only/daemonset.yaml#L111
and https://github.com/just-containers/s6-overlay#container-environment

It is not clear if it set it in an higher level.

Reply:
No need to add it in the other scripts. I will create an issue to for the others scripts.